### ISSUE TITLE
invalid UTF-8 encoding when running shfmt

### ISSUE THREAD

**Issue by rowheel** [termux-play-store/termux-apps#34]
### Problem description

```
$ shfmt
/data/data/com.termux/files/usr/bin/shfmt:1:9: invalid UTF-8 encoding
```

### Steps to reproduce the behavior.

```
$ apt-get install shfmt
$ shfmt
```

### System information

* Termux application version: googleplay.2024.07.07
* Android OS version: 14
* Device model: OnePlus Ace 3

**Comment by fornwall** [comment]
Thanks for reporting!

This should now be fixed in version `3.8.0-1` of the `shfmt` package - can you try updating with `pkg up` and check that it works as expected?

**Comment by rowheel** [comment]
Yes, it's working as expected. Thanks for updating and I'm closing this issue.
